charts/newrelic-infrastructure-v3/values.yaml

```
@@ -36,15 +36,20 @@ common:
   # Configuration entries that apply to all instances of the integration: kubelet, ksm and control plane.
   config:
     # -- How often the integration should collect and report data.
-    # Intervals larger than 40s are not supported and will cause the NR UI to not behave properly.
-    interval: 15s
+    # Intervals larger than 40s are not supported and will cause the NR UI to not behave properly. This value takes precedence over `lowDataMode`
+    # but if this value is undefined its default from 15s (without low data mode) to 30s (with low data mode enabled)
+    # interval: 15s
```

We can hint `helm-docs` about the default value
```suggestion
    # Intervals larger than 40s are not supported and will cause the NR UI to not behave properly.
    # @default -- `15s` if `lowDataMode == false`, `30s` otherwise.
    # interval: 15s
```